source/Ship.cpp

```diff
@@ -2607,13 +2618,32 @@ double Ship::MaxShields() const
 }
 
 
+
 double Ship::MaxHull() const
 {
 	return attributes.Get("hull") * (1 + attributes.Get("hull multiplier"));
 }
 
 
 
+double Ship::ShieldRepairLimit() const
+{
+	double shieldLimit = attributes.Get("nonregenerative shields");
+
+
+	return 1 - shieldLimit;
+}
+
+
+
+double Ship::HullRepairLimit() const
```

Comment:
Seems unnecessary, just name the attributes limit and there is no need for this conversion

Reply:
No no, what I mean is rename the attribute so you dont need to calc

Reply:
Ok